Sources/AccioKit/Services/CachedBuilderService.swift

```diff
@@ -31,7 +31,8 @@ final class CachedBuilderService {
         appTarget: AppTarget,
         dependencyGraph: DependencyGraph,
         platform: Platform,
-        swiftVersion: String
+        swiftVersion: String,
+        toolchain: String? = nil
```

Comment:
Same here?

Reply:
It is nil to fallback to the default behaviour as before to avoid breaking changes